vortex-array/src/arrays/primitive/compute/take.rs

```diff
@@ -72,11 +73,11 @@ register_kernel!(TakeKernelAdapter(PrimitiveVTable).lift());
 /// # Parameters
 /// * `indices` - Indices to gather values from
 /// * `values` - Source values to index
-/// * `nullability` - Nullability of the resulting array
 ///
 /// # Returns
 /// A `PrimitiveArray` containing the gathered values where each index has been replaced with
 /// the corresponding value from the source array.
+#[multiversion(targets("x86_64+avx2", "x86_64+avx", "aarch64+neon"))]
```

Comment:
you could kill the `x86_64+avx` variant. These are cpus produced before 2013? All the modern cpus that don't support avx2 also don't support avx2